### PR TITLE
Refactor FXIOS-5658 [v112] Update localized string from Sites to Pages

### PR DESCRIPTION
**Pull request by PARAIPAN9**

### CODE REVIEW

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lmarceau) [inline .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:1]: Not sure why there's this workspace data change? 🤔

File: Client/Frontend/Strings.swift [hunk 2]
```diff
@@ -179,11 +179,11 @@ extension String {
     /// Identifiers of all new strings should begin with `FirefoxHome.`
     public struct FirefoxHomepage {
         public struct Common {
-            public static let SitesCount = MZLocalizedString(
-                "FirefoxHomepage.Common.SitesCount.v101",
+            public static let PagesCount = MZLocalizedString(
+                "FirefoxHomepage.Common.PagesCount.v101",
```
Comment (lmarceau) [inline Client/Frontend/Strings.swift:183]: Had a discussion with @dnarcese, to give enough time for translations we can merge this change in v112. We can make sure the key reflects that with: 
```suggestion
            public static let PagesCount = MZLocalizedString(
                "FirefoxHomepage.Common.PagesCount.v112",
```

The PR title can also be adapted. We are cutting off the v111 release today so we'll be able to merge this tomorrow.

**Review by lmarceau — APPROVED** [review]

**Comment by lmarceau** [comment]
Hello @PARAIPAN9, it seems the build failed due to a needed adjustment in unit tests within `HistoryHighlightsViewModelTests`:
Screen Shot 2023-02-15 at 11 03 58 AM
Can you have a look? 🙏

**Comment by mte-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Project coverage: 29.55%</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Edited 3 files</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Created 1 files</td>
    </tr>
  </tbody>
</table>

## Client.app: Coverage: 26.23
| File | Coverage ||
| --- | --- | --- |
ASGroup.swift | 48.0% | ⚠️

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 207de5e38dac80ba1237ae8ab27c6a1b8413eca0
</p>

**Comment by lmarceau** [comment]
@isabelrios the Bitrise was aborted on workflow `build_and_test_1_SmokeTest2` multiple times [here](https://app.bitrise.io/app/6c06d3a40422d10f/pipelines/ac95d4e2-d5c8-429d-b84c-ce159caf1a7a). Do you think we can merge as is? I think this isn't related to this PR.

**Pull request merged**